### ISSUE TITLE
ValueError: ids must be list of strings, found: 'docker://zabbix-agent-centos' (type: unicode)

### ISSUE THREAD

**Issue by dj-wasabi** [metacloud/molecule#432]
Hi,

I've installed the latest molecule version, but when running  verify I get the following error:
```
 (master) $ molecule verify
WARNING:vagrant:The Vagrant executable cannot be found. Please check if it is in the system path.
--> Executing ansible-lint.
--> Executing flake8 on *.py files found in tests/.
--> Executing testinfra tests found in tests/.
============================= test session starts ==============================
platform linux2 -- Python 2.7.5, pytest-3.0.2, py-1.4.31, pluggy-0.3.1
rootdir: /git/ansible/ansible-zabbix-agent, inifile:
plugins: cov-2.3.1, xdist-1.14, testinfra-1.4.1
collected 0 items / 1 errorss

==================================== ERRORS ====================================
____________________ ERROR collecting tests/test_docker.py _____________________
/usr/lib/python2.7/site-packages/testinfra/plugin.py:128: in pytest_generate_tests
    "_testinfra_backend", params, ids=ids, scope="module")
/usr/lib/python2.7/site-packages/_pytest/python.py:837: in parametrize
    raise ValueError(msg % (saferepr(id_value), type(id_value).__name__))
E   ValueError: ids must be list of strings, found: 'docker://zabbix-agent-centos' (type: unicode)
============================ pytest-warning summary ============================
WP1 None Modules are already imported so can not be re-written: testinfra
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
================== 1 pytest-warnings, 1 error in 0.41 seconds ==================
ERROR: 

  RAN: '/usr/bin/testinfra tests/test_docker.py --connection=docker'

  STDOUT:


  STDERR:
vagrant@localhost [ /git/ansible/ansible-zabbix-agent -- Fri Sep 02 11:49:23 ]
 (master) $ molecule --version
WARNING:vagrant:The Vagrant executable cannot be found. Please check if it is in the system path.
1.10.0
```

**Comment by retr0h** [comment]
Are you running against an existing role, or a brand new role created with 1.10.0 via `molecule init`?

**Comment by retr0h** [comment]
Any chance you can link me to the repo?

**Comment by retr0h** [comment]
@dj-wasabi Interesting.  I got the same error when I upgraded from pytest-2.9.2 to pytest-3.0.2.   I wonder if testinfra is not pytest 3.0 compatible.

**Comment by retr0h** [comment]
> I wonder if testinfra is not pytest 3.0 compatible.

Ah, yes [that is the issue](https://github.com/philpep/testinfra/issues/125).

**Comment by retr0h** [comment]
Hi @dj-wasabi I uploaded a new package to pypi (1.10.2).  This should correct your issue.  Otherwise install 3.0.3 or newer of pytest, which was recently fixed.

**Comment by retr0h** [comment]
Fixed by [1.10.2](https://pypi.python.org/pypi/molecule).
